Gems/Atom/Feature/Common/Code/Source/RayTracing/RayTracingAccelerationStructurePass.h

```diff
@@ -48,6 +48,12 @@ namespace AZ
 
             // revision number of the ray tracing data when the TLAS was built
             uint32_t m_rayTracingRevision = 0;
+
+            // keeps track of the current frame to determine updates or rebuilds of the skinned BLASes
+            uint32_t m_frameCount = 0;
```

Comment:
Should this be 64 bit as you are not resetting the framecount anywhere?

Reply:
That is a good idea. I have made the necessary changes.